code/modules/mob/living/simple_animal/abnormality/teth/beauty_beast.dm

```diff
@@ -0,0 +1,43 @@
+/mob/living/simple_animal/hostile/abnormality/beauty
+	name = "Beauty and the Beast"
+	desc = "A giant, disgusting creature."
+	icon = 'ModularTegustation/Teguicons/48x32.dmi'
+	icon_state = "beauty"
+	maxHealth = 650
+	health = 650
+	threat_level = TETH_LEVEL
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = list(40, 20, -20, -20, -20),
+		ABNORMALITY_WORK_INSIGHT = list(50, 50, 40, 30, 30),
+		ABNORMALITY_WORK_ATTACHMENT = list(30, 15, -50, -50, -50),
+		ABNORMALITY_WORK_REPRESSION = list(65, 65, 65, 65, 65)
+		)
+	work_damage_amount = 3
```

Comment:
The damage is rather low, isn't it?

Reply:
Right, sorry i just plugged in the original value here